Patterns/Chinese/Chinese-Numbers.yaml

```diff
@@ -234,7 +234,7 @@ DoubleWithMultiplierRegex: !nestedRegex
   def: ({NegativeNumberTermsRegexNum}\s*)?{ZeroToNineFullHalfRegex}+[\.．]{ZeroToNineFullHalfRegex}+\s*{BaseNumbers.NumberMultiplierRegex}
   references: [NegativeNumberTermsRegexNum, ZeroToNineFullHalfRegex, BaseNumbers.NumberMultiplierRegex]
 DoubleWithThousandsRegex: !nestedRegex
-  def: '{NegativeNumberTermsRegex}?{ZeroToNineFullHalfRegex}+([\.．]{ZeroToNineFullHalfRegex}+)?\s*[多几幾余]?[万亿萬億]{1,2}'
+  def: '({NegativeNumberTermsRegex}?{ZeroToNineFullHalfRegex}+([\.．]{ZeroToNineFullHalfRegex}+)?\s*[多几幾余]?[万亿萬億]{1,2})|({NegativeNumberTermsRegex}?{ZeroToNineFullHalfRegex}{1,3}(,{ZeroToNineFullHalfRegex}{3})+([\.．]{ZeroToNineFullHalfRegex}+)?\s*[多几幾余]?[万亿萬億]{1,2})'
```

Comment:
Could this pattern be simplified? It seems to mostly duplicate the previous one. If simplifying it now would be hard, we can use the new one for now, as long as there is no negative performance impact.

Reply:
Thank you for your advice. I've simplified the regex pattern.